### PR TITLE
Translation of tutorial/index

### PR DESCRIPTION
**Pull request by jonasgrunert**
TODO: Spelling Check
Formatting

### CODE REVIEW

File: docs/tutorial/index.md [hunk 1]
```diff
@@ -3,37 +3,71 @@ title: Gatsby.js Tutorials
...
+Lerne mehr über die Nutzung und das Erstellen von Themen mit dem
+[Gatsby Themen Tutorial](/tutorial/theme-tutorials/).
 
-## Additional tutorials
+## Zusätzliche Tutorials
 
-There are many use cases for Gatsby, some of which aren't covered in the first set of tutorials to keep you focused while learning. Gatsby's Additional Tutorials section is a collection of resources showing how to complete even more Gatsby tasks step-by-step, such as using source plugins for images and CMS content.
+Es gibt viele Nutzungsszenarien für Gatsby, einige werden nicht von dieser
+ersten Reihe von Tutorials abgedeckt um dich beim Lernen fokussiert zu halten.
+Gatsbys zuätzliche  Tutorials Sektion ist eine Sammlung von Anleitungen, die
```
Comment (medusalix) [inline docs/tutorial/index.md]: Two spaces 😃.

Comment (medusalix) [inline docs/tutorial/index.md]: - `zuätzliche` should be `zusätzliche`
- replace the two spaces with one

File: docs/tutorial/index.md [hunk 2]
```diff
@@ -3,37 +3,71 @@ title: Gatsby.js Tutorials
...
 
-## Theme tutorials
+## Themen Tutorials
 
-Gatsby themes abstract all of your default configuration (shared functionality, data sourcing, design) out of your site and into an installable package.
+Gatsby Themen abstrahieren alle Standardeinstellungen (geteilte Funktionalität,
+Datenquellen, Design) aus deiner Seite und in ein installierbares Paket.
 
-This means that the configuration and functionality isn’t directly written into your project, but rather versioned, centrally managed, and installed as a dependency. You can seamlessly update a theme, compose themes together, and even swap out one compatible theme for another.
+Das bedeutet das die Konfiguration und Funktionalität nicht direkt in deinem
```
Comment (medusalix) [inline docs/tutorial/index.md]: Das bedeutet, **dass**...

File: docs/tutorial/index.md (hunk 1 above)
Comment (jonasgrunert) [reply docs/tutorial/index.md]: Thanks, must have slipped through

File: docs/tutorial/index.md (hunk 2 above)
Comment (jonasgrunert) [reply docs/tutorial/index.md]: Perfect catch. My bad.

**Comment by muescha** [comment]
i see on this page some translations which are 2/3 words but which i would write more with `-` or in combined words, like:

Blog Beitrage -> Blog-Beiträge -> Blogbeiträge

File: docs/tutorial/index.md [hunk 3]
```diff
@@ -3,37 +3,71 @@ title: Gatsby.js Tutorials
 disableTableOfContents: true
 ---
 
-Welcome to Gatsby! We’re glad you’re here. The goal of this tutorial is to guide you through setting up and deploying your first Gatsby site using a starter template. As we walk through that process, we’ll introduce some more general web development topics, and go over the underlying structure of a Gatsby site.
+Willkommen bei Gatsby! Wir sind froh, dass du hier bist. Das Ziel dieses
+Tutorials ist es dich durch das Aufsetzen und die Entwicklung deiner ersten
+Gatsby Seite basiered auf einer Starter Vorlage zu leiten. Während wir diesen
+Prozess durchführen werden wir einige allgemeinere Webentwicklungsthemen
+einführen und die unterliegende Struktur einer Gastby Seite erläutern.
```
Comment (LekoArts) [inline docs/tutorial/index.md]: Typo here! :) `Gastby`

Comment (muescha) [reply docs/tutorial/index.md]: #6

File: docs/tutorial/index.md [hunk 4]
```diff
@@ -3,37 +3,71 @@ title: Gatsby.js Tutorials
...
-1.  [Get to know Gatsby building blocks](/tutorial/part-one/): Starting new projects, developing, and deploying sites.
-1.  [Introduction to using CSS in Gatsby](/tutorial/part-two/): Explore libraries like Typography.js and CSS Modules.
-1.  [Building nested layouts in Gatsby](/tutorial/part-three/): Layouts are sections of your site that are reused across multiple pages like headers and footers.
+0. [Setz deine eigene Entwicklungsumgebung auf](/tutorial/part-zero/): Wir
+   führen dich in die Kerntechnologien, die Gatsby betreiben, ein und leiten
+   dich durch das Aufsetzen deiner Entwicklungsumgebung.
+1. [Lerne die Gatsby Bausteine kennen](/tutorial/part-one/): Der Beginn neuer
+   Projekte, das Entwickeln und Veröffentlichen von Webseiten.
+2. [Einführung in die Nutzung von CSS in Gatsby](/tutorial/part-two/): Entdecke
+   Biblitheken wie Typography.js und CSS Module.
```
Comment (rwachtler) [inline docs/tutorial/index.md]: Missing the "o" for "Bibliotheken"

File: docs/tutorial/index.md [hunk 5]
```diff
@@ -3,37 +3,71 @@ title: Gatsby.js Tutorials
...
+   Blogs und Nutzung einer GraphQL Anfrage um deinen Webseiten Titel in die
+   Kopfzeile des Blogs zu ziehen.
+5. [Datenquellen Plugins und Darstellung von angefragten Daten](/tutorial/part-five/):
+   Nutzen eines Datenquellen Plugins um Markdown Blog Beiträge in deine Seite
+   zu ziehen und Erstellung einer Index Seite mit der Liste aller Blog Beiträge.
+6. [Umformende Plugins](/tutorials/part-six/): Nutzung eines umformenden
+   Plugins um Markdown Blog Beiträge in eine Form umzuwandeln, die der Blog
+   darstellen kann.
+7. [Programatisches Erstellen von Seiten aus Daten](/tutorial/part-seven/):
+   Lerne wie Seiten für deinen Blog programatisch erstellt werden können.
```
Comment (rwachtler) [inline docs/tutorial/index.md]: Missing a "m" for "programmatisch"

File: docs/tutorial/index.md [hunk 6]
```diff
@@ -3,37 +3,71 @@ title: Gatsby.js Tutorials
...
+8. [Vorbereiten einer Seite zum Veröffentlichen](/tutorial/part-eight/):
+   Lerne wie du deine Seite auf Performance und empfohlene Vorgehensweisen
+   für Barrierefreiheit, SEO und mehr prüfen kannst.
 
-## Theme tutorials
+## Themen Tutorials
 
-Gatsby themes abstract all of your default configuration (shared functionality, data sourcing, design) out of your site and into an installable package.
+Gatsby Themen abstrahieren alle Standardeinstellungen (geteilte Funktionalität,
+Datenquellen, Design) aus deiner Seite und in ein installierbares Paket.
```
Comment (rwachtler) [inline docs/tutorial/index.md]: Despite the "and" is present in english, I'd remove "und" here. Seems a little bit out of place here, if you try to read the whole sentence.

File: docs/tutorial/index.md [hunk 7]
```diff
@@ -3,37 +3,71 @@ title: Gatsby.js Tutorials
...
 
-This means that the configuration and functionality isn’t directly written into your project, but rather versioned, centrally managed, and installed as a dependency. You can seamlessly update a theme, compose themes together, and even swap out one compatible theme for another.
+Das bedeutet, dass die Konfiguration und Funktionalität nicht direkt in deinem
+Projekt fest geschrieben ist, sondern als abhängiges Paket versioniert, zentral
```
Comment (rwachtler) [inline docs/tutorial/index.md]: "festgeschrieben"

File: docs/tutorial/index.md [hunk 8]
```diff
@@ -3,37 +3,71 @@ title: Gatsby.js Tutorials
...
+verwaltet und installiert wird. Du kannst problemlos ein Thema aktualiseren,
```
Comment (rwachtler) [inline docs/tutorial/index.md]: Missing an "i" for "aktualisieren"

File: docs/tutorial/index.md [hunk 9]
```diff
@@ -3,37 +3,71 @@ title: Gatsby.js Tutorials
...
-Learn more about using and building themes with the [Gatsby theme tutorials](/tutorial/theme-tutorials/).
+Lerne mehr über die Nutzung und das Erstellen von Themen mit dem
+[Gatsby Themen Tutorial](/tutorial/theme-tutorials/).
 
-## Additional tutorials
+## Zusätzliche Tutorials
 
-There are many use cases for Gatsby, some of which aren't covered in the first set of tutorials to keep you focused while learning. Gatsby's Additional Tutorials section is a collection of resources showing how to complete even more Gatsby tasks step-by-step, such as using source plugins for images and CMS content.
+Es gibt viele Nutzungsszenarien für Gatsby, einige werden nicht von dieser
+ersten Reihe von Tutorials abgedeckt um dich beim Lernen fokussiert zu halten.
```
Comment (rwachtler) [inline docs/tutorial/index.md]: I'd add a "," after "abgedeckt"

File: docs/tutorial/index.md [hunk 10]
```diff
@@ -3,37 +3,71 @@ title: Gatsby.js Tutorials
 disableTableOfContents: true
 ---
 
-Welcome to Gatsby! We’re glad you’re here. The goal of this tutorial is to guide you through setting up and deploying your first Gatsby site using a starter template. As we walk through that process, we’ll introduce some more general web development topics, and go over the underlying structure of a Gatsby site.
+Willkommen bei Gatsby! Wir sind froh, dass du hier bist. Das Ziel dieses
+Tutorials ist es dich durch das Aufsetzen und die Entwicklung deiner ersten
```
Comment (rwachtler) [inline docs/tutorial/index.md]: Add "," after "ist es"

File: docs/tutorial/index.md [hunk 11]
```diff
@@ -3,37 +3,71 @@ title: Gatsby.js Tutorials
...
+Gatsby Seite basiered auf einer Starter Vorlage zu leiten. Während wir diesen
```
Comment (rwachtler) [inline docs/tutorial/index.md]: Missing "n" for "basierend"

File: docs/tutorial/index.md [hunk 12]
```diff
@@ -3,37 +3,71 @@ title: Gatsby.js Tutorials
...
+Prozess durchführen werden wir einige allgemeinere Webentwicklungsthemen
```
Comment (rwachtler) [inline docs/tutorial/index.md]: Add "," after "durchführen"

File: docs/tutorial/index.md (hunk 4 above)
Comment (muescha) [reply docs/tutorial/index.md]: fixed in #5

File: docs/tutorial/index.md (hunk 5 above)
Comment (jonasgrunert) [reply docs/tutorial/index.md]: Hopefully done in #10

File: docs/tutorial/index.md (hunk 7 above)
Comment (jonasgrunert) [reply docs/tutorial/index.md]: Hopefully done in #10

File: docs/tutorial/index.md (hunk 9 above)
Comment (jonasgrunert) [reply docs/tutorial/index.md]: Hopefully done in #10

File: docs/tutorial/index.md (hunk 10 above)
Comment (jonasgrunert) [reply docs/tutorial/index.md]: Hopefully done in #10

File: docs/tutorial/index.md (hunk 12 above)
Comment (jonasgrunert) [reply docs/tutorial/index.md]: Hopefully done in #10

File: docs/tutorial/index.md (hunk 8 above)
Comment (jonasgrunert) [reply docs/tutorial/index.md]: Hopefully done in #10

File: docs/tutorial/index.md (hunk 6 above)
Comment (jonasgrunert) [reply docs/tutorial/index.md]: Hopefully done in #10

**Pull request merged**